modules/rosa-create-objects.adoc

```diff
@@ -273,6 +273,65 @@ Create a cluster with autoscaling enabled on the default worker machine pool.
 $ rosa create cluster --cluster-name=mycluster -region=us-east-1 --enable-autoscaling --min-replicas=2 --max-replicas=5
 ----
 
+[id="rosa-create-break-glass-credential_{context}"]
+== create break glass credential
+
+Create a break glass credential for a hosted control plane cluster with external authentication enabled.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create break-glass-credential --cluster=<cluster_name> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster <cluster_name>\|<cluster_id>
+|Required. The name or ID of the cluster to which the break glass credential will be added.
+
+| --expiration duration
+|Optional. How long a break glass credential can be used before expiring. The expiration duration must be a minimum of 10 minutes and a maximum of 24 hours. If you do not enter a value, the expiration duration defaults to 24 hours.
+
+|--username
+|Optional. The username for the break glass credential. If you do not enter a value, the username defaults to the name of the user creating the break glass credential.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Specifies an AWS profile (string) from your credentials file.
+
+|--region
+|Specifies an AWS region, overriding the `AWS_REGION` environment variable.
+
+|--yes
+|Automatically answers `yes` to confirm the operation.
+|===
+
+.Examples
+Add a break glass credential to a cluster named `mycluster`.
+
+.Syntax
+[source,terminal]
+----
+$ rosa break-glass-credential --cluster-name=mycluster
```

Comment:
Here the command must be "rosa create break-glass-credential --cluster=mycluster"